impl_test.go

```diff
@@ -186,6 +189,44 @@ func TestFuncs(t *testing.T) {
 				},
 			},
 		},
+		{
+			iface: "error",
+			want: []Func{
+				{
+					Name: "Error",
+					Res:  []Param{{Type: "string"}},
+				},
+			},
+		},
+		{
+			iface: "http.Flusher",
+			want: []Func{
+				{
+					Name:     "Flush",
+					Comments: "// Flush sends any buffered data to the client.\n",
```

Comment:
Will these tests break if the comments change in the main Go repo? Does that potentially mean that we can't have a single set of passing tests across multiple Go versions?

Reply:
Done

Reply:
Yep, it definitely will break if the comments change in the Go repo. Do you think I should try to introduce a test file with interfaces that would reflect all test cases (including comments) that unit tests are currently targeting?

Reply:
That would be better. But in a pinch I guess we can just wait until something breaks and then deal with it when it does.